controllers/drplacementcontrol_controller_test.go

```diff
@@ -1009,4 +1139,168 @@ var _ = Describe("DRPlacementControl Reconciler", func() {
 			})
 		})
 	})
+	Context("DRPlacementControl Reconciler Sync DR", func() {
+		userPlacementRule := &plrv1.PlacementRule{}
+		drpc := &rmn.DRPlacementControl{}
+		When("An Application is deployed for the first time", func() {
+			It("Should deploy to East1ManagedCluster", func() {
+				By("Initial Deployment")
+				userPlacementRule, drpc = InitialDeploymentSync(DRPCNamespaceName, UserPlacementRuleName, East1ManagedCluster)
+				verifyInitialDRPCDeployment(userPlacementRule, drpc, East1ManagedCluster)
+			})
+		})
+		When("DRAction changes to Failover", func() {
+			It("Should not failover to Secondary (East2ManagedCluster) till PV manifest is applied", func() {
+				By("\n\n*** Failover - 1\n\n")
+				restorePVs = false
+				fenceCluster(East1ManagedCluster)
+				setDRPCSpecExpectationTo(rmn.ActionFailover, East1ManagedCluster, East2ManagedCluster)
+				verifyUserPlacementRuleDecisionUnchanged(userPlacementRule.Name, userPlacementRule.Namespace, East1ManagedCluster)
+				Expect(getManifestWorkCount(East2ManagedCluster)).Should(Equal(2)) // MWs for VRG and VRG ROLES
+				Expect(len(userPlacementRule.Status.Decisions)).Should(Equal(0))
+				restorePVs = true
+			})
+			It("Should failover to Secondary (East2ManagedCluster)", func() {
+				By("\n\n*** Failover - 1\n\n")
+				verifyFailoverToSecondary(userPlacementRule, East1ManagedCluster, East2ManagedCluster)
+			})
+		})
+		When("DRAction is set to Relocate", func() {
```

Comment:
Like fencing is done for failver tests,  for relocate tests (both here and in the below tests), unfence needs to be done.